frontend/src/screens/settings/Backup.tsx

```diff
@@ -0,0 +1,460 @@
+                          <AlertDialogContent>
+                            <AlertDialogHeader>
+                              <AlertDialogTitle>
+                                Alby Hub Restart Required
+                              </AlertDialogTitle>
+                              <AlertDialogDescription>
+                                <div>
+                                  <p>
+                                    As part of enabling VSS your hub will be
+                                    shut down, and you will need to enter your
+                                    unlock password to start it again.
+                                  </p>
+                                  <p className="mt-2">
+                                    Please ensure you have no pending payments
+                                    or channel closures before continuing.
+                                  </p>
+                                </div>
+                              </AlertDialogDescription>
+                            </AlertDialogHeader>
+                            <AlertDialogFooter>
+                              <AlertDialogCancel>Cancel</AlertDialogCancel>
+                              <AlertDialogAction
+                                onClick={() => migrateLDKStorage("VSS")}
+                              >
+                                Confirm
+                              </AlertDialogAction>
+                            </AlertDialogFooter>
+                          </AlertDialogContent>
+                        </AlertDialog>
+                      )}
+                    </div>
+                  </>
+                )}
+              </>
+            ) : (
+              <div className="flex flex-col gap-8">
+                <div>
+                  <p className="text-muted-foreground text-sm mb-4">
+                    Link your Alby Account to enable automated channel backups
+                    after each channels creation.
+                  </p>
+                  <Button
+                    type="button"
+                    variant={"secondary"}
+                    className="flex gap-2 justify-center"
+                    onClick={() => navigate("/alby/account")}
+                  >
+                    <Link2Icon className="w-4 h-4 mr-2" />
+                    Link Alby Account to Enable
+                  </Button>
+                </div>
+                <div className="flex flex-col gap-4">
+                  <div className="flex gap-2 mb-1 items-center">
+                    <h3 className="text-sm font-medium">
+                      Manual Channels Backup
+                    </h3>
+                    <Badge variant={"positive"}>Active</Badge>
+                  </div>
+                  <p className="text-muted-foreground text-sm ">
+                    To backup your channels state manually, without Alby Account
+                    linked, follow these steps:
```

Comment:
Let's link to the existing guide instead. Telling someone to go to the working directory which they don't know where it is and is different in each environment is not something we can easily add to Alby Hub.

Reply:
resolved